person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    if @age >= 18 || parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] Please use the `of_age?` in the `can_use_services?` to make the code more organized.

```ruby
def can_use_services? 
    of_age?() || @parent_permission
end
```